script/PowerSimulations_examples/7_inter-regional_network_constraints.jl

```diff
@@ -0,0 +1,87 @@
+# # Operations problems with [PowerSimulations.jl](https://github.com/NREL/PowerSimulations.jl)
+
+# **Originally Contributed by**: Clayton Barrows and Sourabh Dalvi
+
+# ## Introduction
+
+# PowerSimulations.jl supports the construction and solution of relaxed optimal power flow
+# problems (Operations Problems). Operations problems form the fundamental
+# building blocks for [sequential simulations](../../notebook/PowerSimulations_examples/sequential_simulations.ipynb).
+# This example shows how to specify and customize a the mathematics that will be applied to the data with
+# an `OperationsProblemTemplate`, build and execute an `OperationsProblem`, and access the results.
+
+# ## Dependencies
+using SIIPExamples
+
+# ### Modeling Packages
+using InfrastructureSystems
+const IS = InfrastructureSystems
+using PowerSystems
+const PSY = PowerSystems
+using PowerSimulations
+const PSI = PowerSimulations
+using D3TypeTrees
+
+# ### Data management packages
+using Dates
+using DataFrames
+
+# ### Optimization packages
+using JuMP
+using Cbc #solver
+
+# ### Data
+# This data depends upon the [RTS-GMLC](https://github.com/grid-mod/rts-gmlc) dataset. Let's
+# download and extract the data.
+
+rts_dir = SIIPExamples.download("https://github.com/GridMod/RTS-GMLC")
+rts_src_dir = joinpath(rts_dir, "RTS_Data", "SourceData")
+rts_siip_dir = joinpath(rts_dir, "RTS_Data", "FormattedData", "SIIP");
+
+
+# ### Create a `System` from RTS-GMLC data just like we did in the [parsing tabular data example.](../../notebook/PowerSystems_examples/parse_tabulardata.jl)
+rawsys = PSY.PowerSystemTableData(rts_src_dir,
+                                  100.0,
+                                  joinpath(rts_siip_dir,"user_descriptors.yaml"),
+                                  timeseries_metadata_file = joinpath(rts_siip_dir,"timeseries_pointers.json"),
+                                  generator_mapping_file = joinpath(rts_siip_dir,"generator_mapping.yaml"));
+
+sys = System(rawsys; forecast_resolution = Dates.Hour(1));
+
```

Comment:
Instead of using the full RTS-GMLC data set (which gets the full RTS-GMLC system with annual data) with the simpler test data, by replacing this code with:
```julia
pkgpath = dirname(dirname(pathof(SIIPExamples)))
include(joinpath(pkgpath,"test", "PowerSystems_examples", "parse_tabulardata.jl"))
```